src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Successfully fetched data from DB for userId: {}", userId);
```

Comment:
The `TODO` comment suggests adding a log message about successfully fetched data from the DB. This is already implemented with the existing log statement. Ensure that the log message provides all necessary information as per the task requirements.